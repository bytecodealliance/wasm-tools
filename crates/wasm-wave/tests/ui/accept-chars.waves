list-chars(['W', 'A', 'V', 'E', 'ğŸŒŠ']);
list-chars(['\\', '\'', '\"', '\t', '\n', '\r']);
list-chars(['\u{0}', '\u{000000}']);
list-chars(['x', '\u{78}', '\u{000078}']);
list-chars(['â˜ƒ', '\u{2603}', '\u{002603}']);
list-chars(['"', '\"', '\u{22}']);