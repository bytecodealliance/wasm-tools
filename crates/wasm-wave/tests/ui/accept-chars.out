list-chars(vals: ['W', 'A', 'V', 'E', 'ğŸŒŠ'])
list-chars(vals: ['\\', '\'', '\"', '\t', '\n', '\r'])
list-chars(vals: ['\u{0}', '\u{0}'])
list-chars(vals: ['x', 'x', 'x'])
list-chars(vals: ['â˜ƒ', 'â˜ƒ', 'â˜ƒ'])
list-chars(vals: ['\"', '\"', '\"'])
